tests/e2e/specs/BoundaryEventCommonBehaviour.spec.js

```diff
@@ -81,26 +81,44 @@ boundaryEventData.forEach(({ type, nodeType, eventXMLSnippet, eventXMLSnippetWit
 
     it('can stay anchored to task when moving pool', function() {
       configurePool({ x: 300, y: 300 });
+
+      const taskSelector = '.main-paper ' +
+        '[data-type="processmaker.components.nodes.task.Shape"]';
+
+      cy.get(boundaryEventSelector).then($boundaryEvent => {
+        cy.get(taskSelector).then($task => {
+          const boundaryPosition = $boundaryEvent.position();
+          const taskPosition = $task.position();
+          expect(boundaryPosition.top).to.be.closeTo(taskPosition.top, 1);
+          expect(boundaryPosition.left).to.be.closeTo(taskPosition.left, 88);
+        });
+      });
+
       cy.get('[data-test=downloadXMLBtn]').click();
       cy.window()
         .its('xml')
         .then(removeIndentationAndLinebreaks)
         .then(xml => {
-          const boundaryEventPositionXml = 'bpmndi:BPMNShape id="node_3_di" bpmnElement="node_3"><dc:Bounds x="232" y="251"';
           expect(xml).to.contain(eventXMLSnippetWithoutNullAttributes);
-          expect(xml).to.contain(boundaryEventPositionXml);
         });
 
-      moveElementRelativeTo({ x: 400, y: 400 }, 150, 150);
+      moveElementRelativeTo({ x: 400, y: 400 }, 50, 50);
+      waitToRenderAllShapes();
+      cy.get(boundaryEventSelector).then($boundaryEvent => {
+        cy.get(taskSelector).then($task => {
+          const boundaryPosition = $boundaryEvent.position();
+          const taskPosition = $task.position();
+          expect(boundaryPosition.top).to.be.closeTo(taskPosition.top, 1);
+          expect(boundaryPosition.left).to.be.closeTo(taskPosition.left, 88);
+        });
+      });
```

Comment:
Since these lines are the same as the one above, do you think it would make it more readable to extract it out into a function like `testThatBoundaryEventIsCloseToTask`?

Reply:
Can do. I wasn't sure if the 88 would change. Since the one above and the one below aren't the same difference. If it is failing, it is a large difference at least.